src/composition/supergraph/config/full/mod.rs

```diff
@@ -0,0 +1,12 @@
+//! Provides objects related to fully resolving a supergraph config.
+//!
+//! Full resolution is the process of taking a subgraph config and producing
+//! a SDL string from whatever subgraph source is provided
```

Comment:
might be worth calling out the difference between this and lazily resolving? that'd help folks within the file understand its use